cross/libstdc++/Makefile

```diff
@@ -14,60 +14,33 @@ HOMEPAGE = https://gcc.gnu.org/
 COMMENT  = The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Ada, Go, and D, as well as libraries for these languages (libstdc++,...). GCC was originally written as the compiler for the GNU operating system. The GNU system was developed to be 100% free software, free in the sense that it respects the user\'s freedom.
 LICENSE  = GPLv3
 
-CONFIGURE_TARGET = nop
-COMPILE_TARGET = nop
 INSTALL_TARGET = libstdc++6_install
 
 include ../../mk/spksrc.archs.mk
 
-PKG_ARCH = amd64
+# use digests with multiple files
+PKG_DIST_ARCHS = amd64 armhf arm64 armel i386
+
+PKG_DIST_ARCH = amd64
 LIB_DIR = x86_64-linux-gnu
 ifeq ($(findstring $(ARCH),$(ARMv7_ARCHS) $(ARMv5_ARCHS)),$(ARCH))
-PKG_ARCH = armhf
+PKG_DIST_ARCH = armhf
 LIB_DIR = arm-linux-gnueabihf
 else ifeq ($(findstring $(ARCH),$(ARMv8_ARCHS)),$(ARCH))
-PKG_ARCH = arm64
+PKG_DIST_ARCH = arm64
 LIB_DIR = aarch64-linux-gnu
 else ifeq ($(findstring $(ARCH),$(ARMv7L_ARCHS)),$(ARCH))
-PKG_ARCH = armel
+PKG_DIST_ARCH = armel
 LIB_DIR = arm-linux-gnueabi
 else ifeq ($(findstring $(ARCH),$(i686_ARCHS)),$(ARCH))
-PKG_ARCH = i386
+PKG_DIST_ARCH = i386
 LIB_DIR = i386-linux-gnu
 endif
 
-include ../../mk/spksrc.cross-cc.mk
+include ../../mk/spksrc.install-resources.mk
```

Comment:
Would there be any way to revert this and keep using `cross-cc.mk` so from there it calls `install-resoureces.mk`?
That would keep the behavior further aligned with all other packages.
Perhaps even being able to include it in `cross-cmake.mk` and `cross-meson.mk` so it is accessible in all cases?

Reply:
`spksrc.install-resources.mk` was introduced for packages that do not need to comile (i.e. CONFIGURE_TARGET = nop and COMPILE_TARGET = nop) and only need a custom install target.

Reply:
Gotcha, that explains.  Will it also work with other packages using `cross-cc.mk` and al?

Reply:
Yes it works with all cross-*.mk, Only `install-resoureces.mk` was not including `../../mk/spksrc.generate-digests.mk` and had a copy of the previous digests creation code.